docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md

```diff
@@ -26,18 +26,32 @@ For Ubuntu 20.04 LTS ('focal'), use the following file:
 wget https://github.com/savonet/liquidsoap/releases/download/v1.4.4/liquidsoap-v1.4.4_1.4.4-ubuntu-focal-amd64-1_amd64.deb
 ```
 
-For Debian 11 ('Bullseye'), use the following file:
+For Debian 11 ('Bullseye'), use the following files: (Debian 11 ships fdk-aac2, so you'll need to download fdk-aac1 from Debian 10)
 
 ```bash
 wget https://github.com/savonet/liquidsoap/releases/download/v1.4.4/liquidsoap-v1.4.4_1.4.4-debian-stable-amd64-1_amd64.deb
```

Comment:
I believe there is a mistake here, we should use the deb package for `testing` as bullseye 11 was testing at the time of the release of this package. This might be a reason why fdk-aac1 is required, as it would be for the older version of Debian.

Reply:
https://github.com/savonet/liquidsoap/releases/download/v1.4.4/liquidsoap-v1.4.4_1.4.4-debian-testing-amd64-1_amd64.deb

Reply:
Ah, ok, that makes sense. I will make a quick test on the instance i have up and running and alter the docs accordingly.